utils/collect-updates/collect-updates.js

```diff
@@ -43,45 +48,36 @@ function collectUpdates(filteredPackages, packageGraph, execOpts, commandOptions
 
   if (forced.size) {
     // "warn" might seem a bit loud, but it is appropriate for logging anything _forced_
-    log.warn("force-publish", forced.has("*") ? "all packages" : Array.from(forced.values()).join("\n"));
+    const forceLabel = useConventionalGraduate ? "conventional-graduate" : "force-publish";
+    log.warn(forceLabel, forced.has("*") ? "all packages" : Array.from(forced.values()).join("\n"));
   }
 
-  let candidates;
+  // --conventional-commits --conventional-graduate
+  if (useConventionalGraduate) {
+    if (forced.has("*")) {
```

Comment:
If `--conventional-graduate` is different from `--force-publish` in that it only impacts packages that are in prerelease and never affects packages that are not in any way. This is why we don't set all packages as candidates here as we do with `--force-publish`.